src/jaxsim/mujoco/loaders.py

```diff
@@ -203,10 +203,8 @@ def convert(
             joints_dict[joint_name].type = "fixed"
 
         # Convert the ROD model to URDF.
-        urdf_string = rod.urdf.exporter.UrdfExporter.sdf_to_urdf_string(
+        urdf_string = rod.urdf.exporter.UrdfExporter.to_urdf_string(
```

Comment:
Now the options are passed as attributes of the dataclass, can you build the `UrdfExporter` with the same options as before?

Reply:
Thanks, addressed in https://github.com/ami-iit/jaxsim/pull/155/commits/a7bf16d65daa443fb442e62cc191fcbc64115653